amt/models/__init__.py

```diff
@@ -1,5 +1,6 @@
+from .algorithm import Algorithm
 from .project import Project
```

Comment:
Project still exist here (and so does the project.py model). Is that intentional?